packages/x-components/src/components/result/base-result-fallback-image.vue

```diff
@@ -1,12 +1,12 @@
 <template functional>
   <svg
-    :class="['x-picture__image--fallback'].concat(data.staticClass, data.class)"
```

Comment:
I think we should keep here the class attribute:
 
```
    :class="[].concat(data.staticClass, data.class)"
```

This is a functional component, if we do not do that, we will not be able to add classes when using it.